services/core/FailoverAgent/failover/agent.py

```diff
@@ -67,6 +67,7 @@
 
 utils.setup_logging()
 _log = logging.getLogger(__name__)
+__version__ = '0.1'
```

Comment:
Kyle said that if would have thunk about it he would have had you bump it.  You can choose any number you like above 0.1.  I would however suggest 0.2.   But that's just me.  My guide would be http://semver.org/.

I might even be able to argue 0.1.1 as we changed the configuration interface.